advisor/src/test/kotlin/advisors/OssIndexTest.kt

```diff
@@ -123,15 +120,13 @@ class OssIndexTest : WordSpec({
 
             val result = ossIndex.retrievePackageFindings(packages).mapKeys { it.key.id }
 
-            result shouldNotBeNull {
-                keys should containExactly(COMPONENTS_REQUEST_IDS)
-
-                COMPONENTS_REQUEST_IDS.forEach { pkg ->
-                    val pkgResult = getValue(pkg)
-                    pkgResult.advisor shouldBe ossIndex.details
-                    pkgResult.vulnerabilities should beEmpty()
-                    pkgResult.summary.issues shouldHaveSize 1
-                    val issue = pkgResult.summary.issues.first()
+            result.keys shouldBe COMPONENTS_REQUEST_IDS
+            COMPONENTS_REQUEST_IDS.forEach { pkg ->
+                result[pkg] shouldNotBeNull {
+                    advisor shouldBe ossIndex.details
+                    vulnerabilities should beEmpty()
+                    summary.issues shouldHaveSize 1
```

Comment:
can line 128 - 130 be replaced by `shouldHaveSingleElement { it.severity == Severity.ERROR }`?